src/main/java/uk/gov/justice/hmpps/prison/repository/sql/PersonRepositorySql.kt

```diff
@@ -3,8 +3,8 @@ package uk.gov.justice.hmpps.prison.repository.sql
 enum class PersonRepositorySql(val sql: String) {
   GET_PERSON_IDENTIFIERS(
     """
-        select pi.IDENTIFIER_TYPE identifierType,
-        pi.IDENTIFIER identifierValue
+        select pi.IDENTIFIER_TYPE identifier_Type,
```

Comment:
the previous mapper used to check both the java and oracle column names, might need to get the new mapper to do the same, but this is the first occurrence I've seen